hermes/bin/src/runtime_extensions/wasi/io/streams/host.rs

```diff
@@ -32,15 +32,15 @@ impl HostInputStream for HermesRuntimeContext {
     fn read(
         &mut self, _rep: wasmtime::component::Resource<InputStream>, _len: u64,
     ) -> wasmtime::Result<Result<Vec<u8>, StreamError>> {
-        todo!()
+        Ok(Ok(Vec::new()))
```

Comment:
Same as above, add `TODO` comments for functions which need implementing to our internal filesystem logic.